### PR TITLE
Adds Kevinchems, a kevin designed overhaul of chemistry reagents

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

**Comment by imsxz** [comment]
kek

**Comment by kevinz000** [comment]
oranges is drunk

**Comment by RogueSteampunker** [comment]
I'm okay with this

**Comment by ishitbyabullet** [comment]
kev wtf

**Comment by Barhandar** [comment]
Conflicts with #37533 though.

**Comment by Aloraydrel** [comment]
based

**Comment by Ordonis** [comment]
I didn't know Oranges and Kevin coded for vorestation

**Comment by deathride58** [comment]
CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!! CIT REBASE!!!

File: code/modules/reagents/chemistry/reagents/cit_reagents.dm [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+	required_reagents = list("anaphro" = 5, "acetone" = 1)
+	required_temp = 300
+	mix_message = "The mixture thickens and heats up slighty..."
+
+//=========Drinks and Stuff!============//
+
+/datum/reagent/consumable/ethanol/sake
+	name = "Sake"
+	id = "sake"
+	description = "A sweet rice wine of questionable legality and extreme potency."
```
Comment (deathride58) [inline code/modules/reagents/chemistry/reagents/cit_reagents.dm]: Hahaha okay though but when are y'all gonna port this weeb shit

**Comment by iskyp** [comment]
dnm

**Comment by nicbn** [comment]
image

File: code/modules/reagents/chemistry/reagents/cit_reagents.dm (hunk 1 above)
Comment (fludd12) [reply code/modules/reagents/chemistry/reagents/cit_reagents.dm]: ooh! i coded this!

**Comment by Wilchenx** [comment]
add hexacrocin too

**Comment by Shadowflame909** [comment]
re open this is pretty good.

**Comment by Poojawa** [comment]
@PKPenguin321 I've thought about adding a stun to it, but it doesn't currently, so.

Also @optimumtact you silly boy, If you admire Citadel that much you should just code for us. <3

**Comment by Shadowflame909** [comment]
Why did oranges code this only for it to never be used in any way?

**Pull request closed without merging**